editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -2057,6 +2051,7 @@ void Node3DEditorViewport::_sinput(const Ref<InputEvent> &p_event) {
 
 		if (k->get_keycode() == Key::SPACE) {
 			if (!k->is_pressed()) {
+				// TODO: Is this unreachable?
```

Comment:
I noticed this while trying to add a SPACE keybind. If `!k->is_pressed()`, we already return early above, so I don't think it is reachable, or what it is supposed to do.